project/src/main/webapp/event-details.js

```diff
@@ -126,31 +131,34 @@ function getLinkForOpportunity(opportunityId) {
   const urlParams = new URLSearchParams(queryString);
   const eventId = urlParams.get('eventId');
   return `<a href="/update-volunteering-opportunity.html?opportunity-id=${
-    opportunityId}&event-id=${eventId}"
+      opportunityId}&event-id=${eventId}&eventId=${eventId}"
```

Comment:
I'm not sure I understand why there are 2 different query strings for event id, is it possible to fix formatting (might have been done by 2 different people in the past) so that only 1 needs to be passed in?

Reply:
yichen and i used different params - she has using eventId, have used event-id I think i can change my test to remedy this.